CP-CPS.md

```diff
@@ -866,22 +866,19 @@ ISRG Public Keys are also available in the Certificate Repository.
 
 ### 6.1.5 Key sizes
 
-ISRG Root CA RSA Private Keys are at least 4096 bits in length.
+ISRG Root CA keys are either RSA with a modulus of length 4096 or ECDSA with namedCurve P-384.
```

Comment:
This omits "bits" so length is ambiguous. Bytes?

Probably should mirror the BRs:

https://cabforum.org/working-groups/server/baseline-requirements/documents/CA-Browser-Forum-TLS-BR-2.0.7.pdf

> 6.1.5 Key sizes
> For RSA key pairs the CA SHALL:
> • Ensure that the modulus size, when encoded, is at least 2048 bits, and;
> • Ensure that the modulus size, in bits, is evenly divisible by 8.
> For ECDSA key pairs, the CA SHALL:
> • Ensure that the key represents a valid point on the NIST P‐256, NIST P‐384 or NIST P‐521
elliptic curve.

Reply:
Done. I've borrowed the phrasing from the BRs, but kept it as prose (not bulleted).